straxen/plugins/peaklet_processing.py

```diff
@@ -186,6 +186,47 @@ def clip_peaklet_times(peaklets, start, end):
                 p['time'] = start
             if strax.endtime(p) > end:
                 p['length'] = (end - p['time']) // p['dt']
+                
+                
+@export
+@strax.takes_config(
+    strax.Option(
+        'n_tpc_pmts_he',track=False,default=800
```

Comment:
you can use https://github.com/XENONnT/straxen/blob/master/straxen/contexts.py#L25 second argument